Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -1472,6 +1472,24 @@ void CClientVehicle::SetWheelStatus(unsigned char ucWheel, unsigned char ucStatu
     }
 }
 
+//
+// Returns component name for eWheelPosition enum
+//
+const SString CClientVehicle::GetComponentNameForWheel(unsigned char ucWheel) const noexcept
+{
+    switch (ucWheel)
+    {
+        case FRONT_LEFT_WHEEL:
+            return "wheel_lf_dummy";
+        case FRONT_RIGHT_WHEEL:
+            return "wheel_rf_dummy";
+        case REAR_LEFT_WHEEL:
+            return "wheel_lb_dummy";
+        case REAR_RIGHT_WHEEL:
+            return "wheel_rb_dummy";
+    }
```

Comment:
All code-paths must return a value

Reply:
In this case, the function will never get anything other than a wheel, so I don't see any reason to return anything with the default value.

Reply:
A value-returning function must return a value from all code paths; otherwise, it will result in undefined behavior. This is requirements of the standard.